### PR TITLE
[posichain-staking] Add posichain-staking strategy

### PR DESCRIPTION
**Pull request by dannyposi**
Changes proposed in this pull request:
- Add posichain-staking strategy

### CODE REVIEW

File: src/strategies/posichain-staking/index.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+type Params = {
+  symbol: string;
+  decimals: number;
+};
+
+export async function strategy(
+  _space: string,
+  _network: string,
+  provider: StaticJsonRpcProvider,
+  _addresses: Array<string>,
```
Comment (ChaituVR) [inline src/strategies/posichain-staking/index.ts:17]: You were not using _addresses? The response doesn't depend on addresses.

**Review by ChaituVR — CHANGES_REQUESTED** [review]

File: src/strategies/posichain-staking/index.ts (hunk 1 above)
Comment (dannyposi) [reply src/strategies/posichain-staking/index.ts:20]: I've add comment to explain about that. I see strategy caller has filtered out all addresses that not needed: https://github.com/snapshot-labs/snapshot-strategies/blob/02439eb120ed7c4cc0c493924b78d92d22006b40/src/utils.ts#L26

**Review by ChaituVR — APPROVED** [review]

**Comment by dannyposi** [comment]
@ChaituVR Thank for merging. But when it available for choosing?

**Comment by ChaituVR** [comment]
It is live now https://snapshot.org/#/strategy/posichain-staking

**Pull request merged**